### PR TITLE
docs(billing): add route for a single fee schedule

### PR DESCRIPTION
**Pull request by eselman**
… Closes: CB-22

@olmstead , @bondricek , @pablodicosta , @abuecker  please check

### CODE REVIEW

**Comment by pablodicosta** [comment]
:+1:

File: api/v2/advisor/client/billing/examples/single-fee-schedule.res.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (olmstead) [inline api/v2/advisor/client/billing/examples/single-fee-schedule.res.json]: The existing files use camel case and not hyphens in the file names.  We should be consistent. Which is the standard?

File: api/v2/advisor/client/billing/examples/single-fee-schedule.res.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "result" : {
```
Comment (olmstead) [inline api/v2/advisor/client/billing/examples/single-fee-schedule.res.json]: None of the current responses have a result field, all the data is simply at the top level. The also contain "status" and "message" at the same level as the data. We should be consistent either way.

File: api/v2/advisor/client/billing/fees.raml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/fees/{feeId}:
```
Comment (olmstead) [inline api/v2/advisor/client/billing/fees.raml]: Do we want to pass the account id here, and return the current fee schedule for that account?

File: api/v2/advisor/client/billing/fees.raml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  uriParameters:
```
Comment (olmstead) [inline api/v2/advisor/client/billing/fees.raml]: I'm new to RAML, but shouldn't this paramater be associate with the get endpoint only?  i.e.:

/{feeId}
  get:
     ...

**Comment by olmstead** [comment]
The title of this story does not match the the implementation.  The commit message in the jira ticket was wrong (CnP error).   I corrected it there.  Should have been "add route for viewing a single fee schedule".

File: api/v2/advisor/client/billing/schema/single-fee-schedule.res.json [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+      "description": "Calculation method used by client",
+      "type": "string",
+      "enum": [ "ACTUAL_DAILY_BALANCE", "AVERAGE_DAILY_BALANCE", "ENDING_BALANCE", "STARTING_BALANCE" ]
+    },
+    "billingTiers": {
+      "description": "Billing Tiers",
+      "type": "array",
+      "items": { "$ref": "#/definitions/billingTier" }
+    },
+    "invoices": {
```
Comment (olmstead) [inline api/v2/advisor/client/billing/schema/single-fee-schedule.res.json]: The invoices for the client will be returned via the invoices endpoint in a separate request.  I don't think this belongs here.

File: api/v2/advisor/client/billing/fees.raml (hunk 3 above)
Comment (olmstead) [inline api/v2/advisor/client/billing/fees.raml]: Just went over this with the team. Conclusion was we will want both endpoints. Will implement others in another story.  Eva, I am going to go ahead and make the minor naming changes in your absence so we can merge this. Hope you don't mind.

**Pull request merged**